### PR TITLE
adding morph-kgc implementation

### PR DESCRIPTION
**Pull request by dachafra**
If this implementation isn't suitable for the webpage, please let us know where we can contribute with it :-)
Also closing `tr` and `td` tags in the table.

### CODE REVIEW

File: implementations.html [hunk 1]
```diff
@@ -53,52 +53,82 @@ <h1>RDF-star Implementations</h1>
...
-            <td>
-          <tr><td>RDF.ex
+              <a href="https://jena.apache.org/documentation/rdfstar/">documentation</a></td>
+            <td></td>
+          </tr>
+          <tr>
+            <td>Eclipse</td>
+            <td><a href="https://rdf4j.org/documentation/programming/rdfstar/">documentation</a></td>
+            <td></td>
+          </tr>
```
Comment (TallTed) [inline implementations.html:98]:
```suggestion
          </tr>
          <tr>
            <td>Morph-KGC</td>
            <td><a href="https://github.com/oeg-upm/morph-kgc">GitHub</a></td>
            <td><a href="https://w3id.org/kg-construct/rml-star">RML-star spec</a></td>
          </tr>
```

File: implementations.html [hunk 2]
```diff
@@ -53,52 +53,82 @@ <h1>RDF-star Implementations</h1>
...
+          <tr>
+            <td>TopBraid EDG</td>
+            <td><a href="https://www.topquadrant.com/rdf-star/">blog post</a></td>
+            <td><a href="https://w3c.github.io/rdf-star/rdf-star-cg-spec.html#sa-mode-and-pg-mode">PG mode</a> with custom <a href="file:///home/pa/animation/rdf-star/cg-spec/editors_draft.html#annotation-syntax">annotation syntax</a></td>
+          </tr>
+          <tr>
+            <td>Morph-KGC</td>
+            <td><a href="https://github.com/oeg-upm/morph-kgc">GitHub</a></td>
+            <td><a href="https://w3id.org/kg-construct/rml-star">RML-star spec</a></td>
+          </tr>
```
Comment (TallTed) [inline implementations.html:131]:
```suggestion
          </tr>
```

File: implementations.html [hunk 3]
```diff
@@ -53,52 +53,82 @@ <h1>RDF-star Implementations</h1>
...
-              <a href="https://jena.apache.org/documentation/rdfstar/">documentation</a>
-            <td>
-              
-          <tr><td>Eclipse rdf4j
-            <td><a href="https://rdf4j.org/documentation/programming/rdfstar/">documentation</a>
-            <td>
-          <tr><td>RDF.ex
+              <a href="https://jena.apache.org/documentation/rdfstar/">documentation</a></td>
+            <td></td>
+          </tr>
```
Comment (TallTed) [inline implementations.html:93]:
```suggestion
          </tr>
```

File: implementations.html [hunk 4]
```diff
@@ -53,52 +53,82 @@ <h1>RDF-star Implementations</h1>
...
-            <td><a href="http://graphdb.ontotext.com/documentation/9.2/free/devhub/rdf-sparql-star.html">documentation</a>
-            <td>
-          <tr><td>Apache Jena
+                in the works</td>
+          </tr>
+          <tr>
+            <td>AnzoGraph</td>
+            <td><a href="https://docs.cambridgesemantics.com/anzograph/v2.2/userdoc/lpgs.htm?Highlight=rdf">documentation</a></td>
+            <td><a href="https://w3c.github.io/rdf-star/rdf-star-cg-spec.html#sa-mode-and-pg-mode">PG mode</a></td>
+          </tr>
```
Comment (TallTed) [inline implementations.html:66]:
```suggestion
          </tr>
          <tr>
            <td>Apache Jena</td>
            <td>
              <a href="https://w3c.github.io/rdf-star/reports/#subj_Apache_Jena_">implementation report</a>,
              <a href="https://jena.apache.org/documentation/rdfstar/">documentation</a></td>
            <td></td>
          </tr>
```

**Review by TallTed — CHANGES_REQUESTED** [review]
Implementations should be listed in alpha order.

(One more reason to make PRs focus on one change at a time....)

**Comment by gkellogg** [comment]
I agree with @afs; please add a commit which makes a minimal change to the original file, simply adding the new implementation without reformatting the existing code.

A separate PR can deal with any necessary reformatting or reordering of entries.

**Comment by dachafra** [comment]
Done, and sorry for the noise

**Review by pchampin — APPROVED** [review]

**Comment by afs** [comment]
Hi @dachafra - looks good as the combined effect.

Could you squash the PR to one commit for the change? The git history will look better.

It's showing up to me as all the changes, then the reverse, in 6 commits.

**Comment by pchampin** [comment]
@afs github allows us to do this at merge time ("squash and merge"). This might be the easiest solution.
@dachafra note however that if we do that your 'main' branch will diverge from our 'main' branch... As a general advice for the future: always create dedicated branches for your pull requests to other repositories, and keep synced branches clean of any change on your side.

**Comment by afs** [comment]
@pchampin yes, but.

It is clearer who the contributor is if the OP does the squash and force-pushes.

It's a detail to keep the history clean, not a blocker.

**Pull request closed without merging**